spec/cli_spec.rb

```diff
@@ -13,17 +13,17 @@
       subject.call
     end
 
-    it 'takes in a text argument displays text' do
-      text = 'test'
-
-      expect(subject).to receive(:display_text).with(text).and_return(text)
-
-      subject.display_text(text)
-    end
-
-    it 'displays welcome message and menu options' do
+    # it 'takes in a text argument displays text' do
+    #   text = 'test'
+    #
+    #   expect(subject).to receive(:display_text).with(text).and_return(text)
+    #
+    #   subject.display_text(text)
+    # end
```

Comment:
I generally don't encourage leaving commented out code in the code base. You can always use git history to retrieve code you removed.

Reply:
commit 0c1e800099ce009f116cafa4cc647171651745ac